tests/system/config/apm-server.yml.j2

```diff
@@ -151,8 +160,15 @@ output.elasticsearch:
   # Enable debug output for selected components.
   #selectors: []
 
+{% if logging_json %}
+  # Set to true to log messages in json format.
+  json: {{ logging_json|default("false") }}
```

Comment:
Since this is enclosed by the `{% if logging_json %}` I don't think the `default("false")` is ever reached. Same for the `logging_level` below.

Reply:
true, i'll update, thanks